### PR TITLE
Add license

### LINKED ISSUE #14: SDK needs a license file

### PR DESCRIPTION
**Pull request by rumpl**
Fixes #14

Signed-off-by: Djordje Lukic <djordje.lukic@docker.com>

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Docker cli
```
Comment (chris-crone) [inline NOTICE]: Copy pasta?

```suggestion
Docker CLI JavaScript SDK
```

**Review by chris-crone — APPROVED** [review]
One minor change then LGTM

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2020 Docker, Inc.
```
Comment (thaJeztah) [inline LICENSE]: This one should not be filled in, but stay as-is. This is just an example section describing "how to apply the license" 

```suggestion
   Copyright [yyyy] [name of copyright owner]
```

See https://www.apache.org/licenses/LICENSE-2.0#apply

Comment (rumpl) [reply LICENSE]: Are you sure? The page says:

```
To apply the Apache License to specific files in your work, attach the following boilerplate declaration, with the fields enclosed by brackets "[]" replaced with your own identifying information.
```

[The cli did add the years and owner](https://github.com/docker/cli/blob/master/LICENSE#L179)

Comment (thaJeztah) [reply LICENSE]: Yes, it's exactly how it's described there; to apply the license,

- take the boilerplate below
- put it in the header of your files
- replace the placeholders with the right information.

The license file itself should not be modified

**Review by ndeloof — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

@justincormack did we want to keep the copyright `Docker, Inc.` or want to change too `The Authors` or something along those lines?

**Comment by justincormack** [comment]
"The Authors" is better.

**Review by justincormack — APPROVED** [review]

**Comment by justincormack** [comment]
I dont know why this has a bunch of formatting changes as well but LGTM

**Pull request merged**